src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -203,6 +203,11 @@ export default class FiatConnectQuote extends NormalizedQuote {
     return this.quote.quote.cryptoAmount
   }
 
+  getCryptoTypeString(): string {
+    // todo eventually rename this to getCryptoType and rename getCryptoType to getCryptoCurrency
```

Comment:
I didn't do the rename in this PR because getCryptoType is actually used in many places, so it added a lot of clutter to the review